readme.md

```diff
@@ -2,6 +2,9 @@
 
 > Yeoman generator for AngularJS - lets you quickly set up a project with sensible defaults and best practices.
 
+This is one of many starting points for building a new single-page app.  For a more complete list of the options and their trade-offs,
```

Comment:
`There are many starting points for building a new Angular single page app, in addition to this one. To see a comparison of the popular options, have a ...`